graph, benchdnn: support implicit causal mask in graph API (#2330)
oneapi-src/oneDNN: doc/graph/operations/GreaterEqual.md

GreaterEqual{#dev_guide_op_greaterequal}

> ================================

Reviewer: ```suggestion
========================================
```

GreaterEqual{#dev_guide_op_greaterequal}
========================================

> ## General

Reviewer 2: ```suggestion
## Overview
```

> GreaterEqual operation performs an element-wise greater-than-or-equal

Reviewer 2: ```suggestion
The GreaterEqual operation performs an element-wise greater-than-or-equal
```

comparison between two given tensors. This operation applies

> multi-directional broadcast rules to ensure compatibility between

Reviewer 2: ```suggestion
the multi-directional broadcast rules to ensure compatibility between
```

> tensors of different shapes.

Reviewer 2: ```suggestion
the tensors of different shapes.
```

\f[ dst = \begin{cases} true & \text{if}\ src_0 \ge src_1 \\
    false & \text{if}\ src_0 < src_1 \end{cases} \f]

> ## Operation attributes

Reviewer 2: ```suggestion
## Operation Attributes
```

| Attribute Name                                               | Description                                                | Value Type | Supported Values         | Required or Optional |
|:-------------------------------------------------------------|:-----------------------------------------------------------|:-----------|:-------------------------|:---------------------|
| [auto_broadcast](@ref dnnl::graph::op::attr::auto_broadcast) | Specifies rules used for auto-broadcasting of src tensors. | string      | `none`,`numpy` (default) | Optional             |

> ## Execution arguments

Reviewer 2: ```suggestion
## Execution Arguments
```

> ### Inputs

Reviewer 2: ```suggestion
### Input
```

| Index | Argument Name | Required or Optional |
|:------|:--------------|:---------------------|
| 0     | `src_0`       | Required             |
| 1     | `src_1`       | Required             |

@note Both src shapes should match and no auto-broadcasting is allowed if

> `auto_broadcast` attributes is `none`. `src_0` and `src_1` shapes can be

Reviewer 2: ```suggestion
the `auto_broadcast` attribute is `none`. `src_0` and `src_1` shapes can be
```

> different and auto-broadcasting is allowed if `auto_broadcast` attributes is

Reviewer 2: ```suggestion
different and auto-broadcasting is allowed if the `auto_broadcast` attribute is
```

> `numpy`. Broadcasting is performed according to auto_broadcast value.

Reviewer 2: ```suggestion
`numpy`. Broadcasting is performed according to `auto_broadcast` value.
```

> ### Outputs

Reviewer 2: ```suggestion
### Output
```

> is `numpy`. Broadcasting is performed according to `auto_broadcast` value.

Reviewer 2: ```suggestion
is `numpy`. Broadcasting is performed according to the `auto_broadcast` value.
```

| Index | Argument Name | Required or Optional |
|:------| --------------|:---------------------|
| 0     | `dst`         | Required             |

> ## Supported data types

Reviewer 2: ```suggestion
## Supported Data Types
```

> GreaterEqual operation supports the following data type combinations.

Reviewer 2: ```suggestion
The GreaterEqual operation supports the following data type combinations.
```

Author: Thank you for your suggestion, Ranu. I've updated the documents and would appreciate it if you could review them again. 😄
